vast/integration/vast-pipelines.yaml

```diff
@@ -5,39 +5,23 @@ vast:
     zeek:
       disable-timestamp-tags: true
   pipelines:
-    remove_flow:
-      - drop:
-          fields:
-            - flow_id
-    remove_action:
-      - drop:
-          fields:
-            - alert.action
-    replace_rrname:
-      - replace:
-          fields:
-            dns.rrname: "foobar.net"
-    select_row:
-      - where:
-          expression: "dest_port > 53"
-    hash_uid:
-      - hash:
-          field: fileinfo.filename
-          out: filename_hash
-    select_fields:
-      - select:
-          fields:
-            - ip
-            - alert.category
-            - timestamp
-            - alert.action
-    pseudonymize_ips:
-      - pseudonymize:
-          method: crypto-pan
-          seed: 8009ab3a605435bea0c385bea18485d8b0a1103d6590bdf48c968be5de53836e
-          fields:
-            - src_ip
-            - dest_ip
+    remove_flow: |
```

Comment:
I realize that every YAML-denoted pipeline in this PR starts with a separator. Do they have to?

Reply:
See [yaml-multiline.info](https://yaml-multiline.info). It's just one way to define a multiline string in YAML.

Reply:
I have amazingly switched up the `|` responsibilties the YAML and our pipeline syntax then. :)